Model/Sales/Repository/PackageRepository.php

```diff
@@ -369,7 +369,7 @@ private function getValueFromAttribute($connection, string $tableName, string $a
     private function setWeightFromOneQuoteProduct($product)
```

Comment:
Kan hier een type in?

Reply:
product is een object en is pas mogelijk vanaf php 7.2